src/components/main-view/main-view.jsx

```diff
@@ -44,40 +47,46 @@ export const MainView = () => {
     });
   }, [token]);
 
-  if (!user) {
-    return (
-    <>
-      <LoginView 
-        onLoggedIn={handleOnLoggedIn}
-      />
-      or
-      <SignupView />
-    </>
-    );
-  }
-
-  if (selectedMovie) {
-    return (
-      <MovieView movie={selectedMovie} onBackClick={() => setSelectedMovie(null)} />
-    );
-  }
-
-if (movies.length === 0) {
-  return <div>The list is empty!</div>;
-}
-
 return (
-  <div>
-    <button onClick={() => { setUser(null); setToken(null); localStorage.clear(); }}>Logout</button>
-    {movies.map((movie) => (
-      <MovieCard 
-      key={movie.id}
-      movie={movie}
-      onMovieClick={(newSelectedMovie) => {
-        setSelectedMovie(newSelectedMovie);
-      }}
-      />
-    ))}
-  </div>
+  <Row className="justify-content-md-center">
+    {!user ? (
+      <Col md={6}>
+        <LoginView
+          onLoggedIn={handleOnLoggedIn}
+        />
+        <div style={{textAlign: "center", fontVariant: "all-small-caps"}}>or</div>
+        <SignupView />
+      </Col>
+    ) : selectedMovie ? (
+      <Col md={10}>
+        <MovieView 
+          movie={selectedMovie} 
+          onBackClick={() => setSelectedMovie(null)}
+        />
+      </Col>
+    ) : movies.length === 0 ? (
+      <div>The list is empty!</div>
+    ) : (
+      <>
+        <Button className="mb-3 mt-3 primaryButton" variant="primary" onClick={() => { 
```

Comment:
`const handleOnLogout = () => {...}`

Inline logic is fine, but when you have this much reusable clearing code you should move it to a handler